### DISCUSSION TITLE
Welcome aboard, Carlo!

### DISCUSSION THREAD

**Discussion by robbyrussell** [ohmyzsh/ohmyzsh#11401] [Show and tell]
👏🎉👏🎉👏🎉👏🎉👏🎉👏🎉👏🎉👏🎉

https://user-images.githubusercontent.com/257/208510553-c8962cd2-6fc8-49f4-b7ae-85d5fc467025.mp4

If you've reported bugs, participated in discussions, or sent a pull request to Oh My Zsh in the last year and a half, this is unlikely to surprise you.

With great pleasure, [Marc](https://github.com/mcornella) and I would like to announce that [Carlo Sala](https://github.com/carlosala) was recently promoted to an official [Level 4/ Maintainer](https://github.com/ohmyzsh/ohmyzsh/wiki/Volunteers#level-4--maintainer) of the Oh My Zsh project.

[Carlo's contributions](https://github.com/ohmyzsh/ohmyzsh/commits?author=carlosala) have been instrumental in helping us review issues, pull requests, and plan our project's future.

**Let's all take a moment to share some appreciation for Carlo!**

👏🎉👏🎉👏🎉👏🎉👏🎉👏🎉👏🎉👏🎉

Thanks,
Robby & Marc

If you're curious about our Volunteer / Tour of Duty, take a look at the following:

- https://github.com/ohmyzsh/ohmyzsh/wiki/Volunteers

**Comment by robbyrussell** [comment]
The website has been updated, too!

![image](https://user-images.githubusercontent.com/257/208511116-b18ccb83-e19b-4243-9727-5298ba8a2903.png)

**Comment by mcornella** [comment]
Thank you for your hard work Carlo, you are the biggest driver for this project to keep going. Congratulations! ✨🎊

**Reply by carlosala** [reply]
Thanks Marc, I really learnt a lot about you (and what's still missing!)
Gràcies ❤️

**Comment by carlosala** [comment]
Thanks Robby! For me it was an absolute pleasure to help improving and maintaining (or, at least, trying to 😊) this amazing project.
So glad to be part of this team 😉
